applications/StructuralMechanicsApplication/tests/SmallTests.py

```diff
@@ -52,6 +52,9 @@ class DynamicBossakTests(StructuralMechanichsTestFactory):
 class DynamicNewmarkTests(StructuralMechanichsTestFactory):
     file_name = "dynamic_test/dynamic_newmark_test"
 
+class DynamicNodalDampingTests(StructuralMechanichsTestFactory):
```

Comment:
Hi Riccardo,
thanks for the link, I will rewrite my test according to the patch test.